services/audit-service/src/main/java/datawave/microservice/audit/health/rabbit/config/RabbitHealthProperties.java

```diff
@@ -0,0 +1,366 @@
+package datawave.microservice.audit.health.rabbit.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+import java.util.Map;
+
+@ConfigurationProperties(prefix = "audit.health.rabbit")
+public class RabbitHealthProperties {
+    private boolean enabled = false;
+    
+    // default - 5 minutes
+    private long healthyPollIntervalMillis = 300000L;
```

Comment:
It might be more readable with `TimeUnit.MINUTES.toMillis(5)`.

Reply:
Sure, updated!